steps/start/independence/template.html

```diff
@@ -25,10 +25,12 @@
             <form action="{{ path if path else url }}" method="post" class="form">
 
                 {% block fields %}
-                    {% if allowUC and benefitType == 'UC' %}
+                    {% if isBenefitEnabled('allowUC') and benefitType == 'UC' %}
```

Comment:
As UC benefit switched on now, can we not just remove this?

Reply:
Done